iron-fit-behavior.html

```diff
@@ -501,29 +491,35 @@
     },
 
 
-    __getPosition: function(hAlign, vAlign, size, positionRect, fitRect) {
+    __getPosition: function(hAlign, vAlign, rect, positionRect, fitRect) {
+      var margin = this._fitInfo.margin;
+      // Consider the margin as part of the size for position calculations.
+      var size = {
```

Comment:
It introduces the dependency on `_fitInfo` being computed already but this is only called in `position`, which requires `_fitInfo` anyways... meh.